nebula_hw_interfaces/include/nebula_hw_interfaces/nebula_hw_interfaces_hesai/hesai_cmd_response.hpp

```diff
@@ -21,6 +21,7 @@
 
 #include <array>
 #include <cstdint>
+#include <iomanip>
```

Comment:
needed?

Reply:
Yes, it supplies `std::setw` and `std::setfill` used below.

Reply:
Yes, it supplies `std::setw` and `std::setfill` used below.

Reply:
Weird that is added in this PR though since the rest of the file was untouched